watertap/examples/flowsheets/case_studies/groundwater_treatment/groundwater_treatment.py

```diff
@@ -199,124 +348,111 @@ def add_costing(m):
 
     m.fs.costing.add_electricity_intensity(m.fs.feed.properties[0].flow_vol)
 
-    m.fs.costing.spent_media_recovery_mass = Expression(
+    m.fs.costing.annual_water_inlet = Expression(
         expr=m.fs.costing.utilization_factor
         * pyunits.convert(
-            m.fs.byproduct.flow_mass_comp[0, "filtration_media"],
-            to_units=pyunits.kg / m.fs.costing.base_period,
-        ),
-        doc="Mass of spent filtration media generated per year (accounting for utilization factor)",
+            m.fs.feed.properties[0].flow_vol,
+            to_units=pyunits.m**3 / m.fs.costing.base_period,
+        )
     )
 
-    m.fs.costing.cost_spent_media_disposal = Expression(
-        expr=pyunits.convert(
-            m.fs.costing.spent_media_recovery_mass
-            * m.fs.costing.filtration_media_disposal_cost,
-            to_units=m.fs.costing.base_currency / m.fs.costing.base_period,
-        ),
-        doc="Disposal cost of spent filtration media per year (accounting for utilization factor)",
+    m.fs.costing.add_LCOW(m.fs.filtered_water.properties[0].flow_vol)
+
+
+def display_costing(m):
+    print("\n+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+    print("++++++++++++++++++++ DISPLAY COSTING ++++++++++++++++++++")
+    print("+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+
+    print("\n---------- Levelized costs ----------")
+    LCOW = value(
+        pyunits.convert(m.fs.costing.LCOW, to_units=pyunits.USD_2020 / pyunits.m**3)
     )
+    print(f"Levelized cost of water: {LCOW:.2f} $/m^3")
 
-    m.fs.costing.fresh_media_mass = Expression(
-        expr=m.fs.costing.utilization_factor
-        * pyunits.convert(
-            m.fs.feed.flow_mass_comp[0, "filtration_media"],
-            to_units=pyunits.kg / m.fs.costing.base_period,
-        ),
-        doc="Mass of fresh filtration media used per year (accounting for utilization factor)",
+    print("\n------------- Capital costs -------------")
+    DCC_normalized = value(
+        pyunits.convert(
+            (m.fs.pump.costing.capital_cost + m.fs.micbatt.costing.capital_cost)
+            / m.fs.costing.TIC
```

Comment:
Is this needed here? It's not clear whether we intended to apply a TIC factor, or if we just wanted a default value of TIC = 2 and would only apply in the case that the provided cost relationship called for its inclusion.

Reply:
I copied this from `metab.py` - @TimBartholomew can you weigh in?

Reply:
Nevermind- this is representative of the practical investment factor for the whole system. In the watertap costing package this is referred to as `factor_total_investment` and in this costing packing (zero order package) we use `TIC`. 

So we're good to go on this.